semantic-python/src/Language/Python/Tags.hs

```diff
@@ -90,7 +91,7 @@ instance ToTagsBy 'Custom Py.Call where
   tags' t@Py.Call{} = gtags t
 
 docComment :: Source -> (Py.CompoundStatement :+: Py.SimpleStatement) Loc -> Maybe Text
-docComment src (R1 (Py.ExpressionStatementSimpleStatement (Py.ExpressionStatement { extraChildren = L1 (Py.PrimaryExpressionExpression (Py.StringPrimaryExpression Py.String { ann })) :|_ }))) = Just (toText (slice src (byteRange ann)))
+docComment src (R1 (Py.SimpleStatement (Prj Py.ExpressionStatement { extraChildren = L1 (Prj (Py.Expression (Prj (Py.PrimaryExpression (Prj Py.String { ann }))))) :|_ }))) = Just (toText (slice src (byteRange ann)))
```

Comment:
I really like how this turned out, with `Py.SimpleStatement` as the constructor for the sum instead of all of those `BlahThingThing` constructor names.